sys-auth/google-oslogin/google-oslogin-20180611.ebuild

```diff
@@ -0,0 +1,45 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+DESCRIPTION="Components to support Google Cloud OS Login. This contains bits that belong in USR"
+HOMEPAGE="https://github.com/GoogleCloudPlatform/compute-image-packages/tree/master/google_compute_engine_oslogin"
+SRC_URI="https://github.com/GoogleCloudPlatform/compute-image-packages/archive/${PV}.tar.gz"
+
+LICENSE="Apache-2.0"
+SLOT="0"
+KEYWORDS="amd64"
+IUSE=""
+
+inherit pam
+
+DEPEND="
+	net-misc/curl[ssl]
+	dev-libs/json-c
+	virtual/pam
+"
+RDEPEND="${DEPEND}"
+
+# redefined later after patching
+S=${WORKDIR}/compute-image-packages-${PV}/
+
+src_prepare() {
+	# We only care about the oslogin part of the repo, but patches are generally
+	# generated for the repo as a whole. Do the patching then change $S to be the
+	# directory we actually want.
+	eapply "$FILESDIR/0001-pam_module-use-var-lib-instead-of-var.patch"
+	S=${WORKDIR}/compute-image-packages-${PV}/google_compute_engine_oslogin
+	default
+}
+
```

Comment:
Upon further inspection, this project is yet another non-standard build system, and so will need to inherit `toolchain-funcs` and add a `src_compile` to use dependencies from the board (unless you want to start installing `json-c` and `pam` in the SDK root):

```
src_compile() {
	emake CC="$(tc-getCC)" CXX="$(tc-getCXX)" JSON_INCLUDE_PATH="${ROOT%/}/usr/include/json-c"
}
```